BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -139,7 +139,7 @@
   gPlatformModuleTokenSpaceGuid.PcdVerInfoBase            | 0xFF000000 | UINT32 | 0x20000190
   gPlatformModuleTokenSpaceGuid.PcdCfgDataLoadSource      | 0x00000002 | UINT32 | 0x20000194
   gPlatformModuleTokenSpaceGuid.PcdPayloadReservedMemSize | 0x00004000 | UINT32 | 0x20000195
-
+  gPlatformModuleTokenSpaceGuid.PcdSmmRebaseEnabled       | 0x00000000 | UINT8  | 0x20000196

```

Comment:
Please rename it to PcdSmmRebaseMode to differentiate with other enable/disable feature PCD.

Reply:
Renamed the PCD in new patch.